recipes/solexaqa/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "solexaqa" %}
+{% set version = "3.1.7.1" %}
+{% set sha256 = "b3bdabf955387a4f6176dbdfda1cf03bb869c077b4eec152304f2e76be5f0cf6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [osx or win]
+
+source:
+  url: https://downloads.sourceforge.net/project/solexaqa/src/SolexaQA++_v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - gxx_linux-64 >=7.3.0
```

Comment:
Please don't do this. Instead, overwrite the compiler config by using conda_build_config.yaml as shown here: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/salmon/conda_build_config.yaml

Reply:
Done